src/ludamus/templates/crowd/user/connected.html

```
@@ -60,23 +60,33 @@ <h6 class="flex items-center gap-1.5 font-semibold text-sm mb-3">
-                                                <button type="button"
+                                        <div class="flex gap-1">
+                                            <button type="button"
+                                                    class="icon-btn icon-btn"
+                                                    onclick="document.getElementById('display-{{ forloop.counter }}').style.display='none'; document.getElementById('edit-{{ forloop.counter }}').style.display='block'">
+                                                {% icon "pencil" class="w-3.5 h-3.5" %}
+                                                <span class="sr-only">{% translate "Edit" %}</span>
+                                            </button>
+                                            <form method="post"
+                                                  action="{% url 'web:crowd:profile-connected-users-claim-link' slug=connected.user.slug %}">
+                                                {% csrf_token %}
+                                                <button type="submit"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)